cms/models/pluginmodel.py

```diff
@@ -290,20 +290,32 @@ def reload(self):
         return CMSPlugin.objects.get(pk=self.pk)
 
     def _get_descendants_count(self):
-        cursor = CMSPlugin._get_database_cursor('write')
-        sql = _get_descendants_cte() + '\n'
-        sql += 'SELECT COUNT(*) FROM descendants;'
-        sql = sql.format(connection.ops.quote_name(CMSPlugin._meta.db_table))
-        cursor.execute(sql, [self.pk])
-        return cursor.fetchall()[0][0]
+        return len(self._get_descendants_ids())
```

Comment:
This is much less efficient, I'd rather keep the previous query and see how it can be adapted for other dbs